### PR TITLE
Tracking system, accessing the CTB

### PR DESCRIPTION
**Pull request by simotae14**
### What does it do?

Added the event <i>didNavigate</i> with the From and to properties when we click in a menu link inside the LeftMenu

### Why is it needed?

As a Strapi PM, I want the access to the CTB to be tracked through the event <i>didGoToCTB</i>, so that analytics can be created on the funnel starting from the moment in which a user accesses the CTB.

### How to test it?

As a Strapi PM
 - when I access the content type builder section from the side menu
   - then I want the didGoToCTB to be fired 
 - when I create a graph on Amplitude
   - then I want the didGoToCTB to be available and working

### CODE REVIEW

File: packages/core/admin/admin/src/components/LeftMenu/index.js [hunk 1]
```diff
@@ -125,7 +126,23 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
               const Icon = link.icon;
 
               return (
-                <NavLink as={RouterNavLink} to={link.to} key={link.to} icon={<Icon />}>
+                <NavLink
+                  as={RouterNavLink}
+                  to={link.to}
+                  key={link.to}
+                  icon={<Icon />}
+                  isActive={(match, location) => {
```
Comment (joshuaellis) [inline packages/core/admin/admin/src/components/LeftMenu/index.js:134]: won't this fire every render?

**Review by joshuaellis — CHANGES_REQUESTED** [review]
Any known reason you've had to update all the snapshots? 🤔

File: packages/core/admin/admin/src/components/LeftMenu/index.js (hunk 1 above)
Comment (gu-stav) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: Also: what does this number tells you in the end? Wouldn't it make more sense to track the pageviews of the CTB?

Comment (simotae14) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: The PM wants to track every click in the Content-Type-Builder link inside the LeftMenu

Comment (marcoautiero) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: It's not simply a matter of counting the clicks on the CTB link in the sidebar, but having a starting point to track the funnels of users interacting with the CTB. It already exists for other items in the sidebar, like didGoToMarketplace.

Comment (gu-stav) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: @marcoautiero Ok, makes sense. What do you think about adding a generic event then, which tracks a navigation in the menu and sends the page as payload? This way we have data for all interactions and don't have to introduce different events ...

Comment (marcoautiero) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: Yep, I thought about it, a `didNavigate` with `from` and `to` properties

Comment (gu-stav) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: I think I'd prefer that if that is an option :)

Comment (joshuaellis) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: I agree

Comment (marcoautiero) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: Making a check with Aurélien

Comment (marcoautiero) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: I received Aurélien approval on the `didNavigate`, sorry Simone, we need to change the description of the User story and the code :) But the future generations will be grateful to us u.u

Comment (simotae14) [reply packages/core/admin/admin/src/components/LeftMenu/index.js:134]: ok sure, can you please also update the ticket with the new requirements, thanks

**Review by joshuaellis — APPROVED** [review]

**Review by gu-stav — DISMISSED** [review]

**Review by joshuaellis — DISMISSED** [review]

**Review by marcoautiero — APPROVED** [review]
For me it's fine

**Review by joshuaellis — APPROVED** [review]

**Pull request merged**